src/Epic/contrib.index.tsx

```diff
@@ -0,0 +1,217 @@
+`;
+
+const highlightWrapperStyles = css`
+    ${body.medium({ fontWeight: 'bold' })}
+    ${linkStyles}
+`;
+
+const highlightStyles = css`
+    padding: 2px;
+    background-color: ${palette.brandAlt[400]};
+`;
+
+type HighlightedProps = {
+    highlightedText: string;
+};
+
+type BodyProps = {
+    paragraphs: string[];
+    highlightedText?: string;
+    countryCode?: string;
+};
+
+interface EpicHeaderProps {
+    text: string;
+}
+
+const EpicHeader: React.FC<EpicHeaderProps> = ({ text }: EpicHeaderProps) => {
+    return <h2 css={headingStyles}>{text}</h2>;
+};
+
+const Highlighted: React.FC<HighlightedProps> = ({ highlightedText }: HighlightedProps) => {
+    return (
+        <strong css={highlightWrapperStyles}>
+            {' '}
+            <span css={highlightStyles}>{highlightedText}</span>
+        </strong>
+    );
+};
+
+interface EpicBodyParagraphProps {
+    paragraph: string;
+    highlighted: JSX.Element | null;
+}
+
+const EpicBodyParagraph: React.FC<EpicBodyParagraphProps> = ({
+    paragraph,
+    highlighted,
+}: EpicBodyParagraphProps) => {
+    return (
+        <p css={bodyStyles}>
+            {paragraph}
+            {highlighted ? highlighted : null}
+        </p>
+    );
+};
+
+const EpicBody: React.FC<BodyProps> = ({ paragraphs, highlightedText }: BodyProps) => {
+    return (
+        <>
+            {paragraphs.map((paragraph, idx) => {
+                const paragraphElement = (
+                    <EpicBodyParagraph
+                        key={idx}
+                        paragraph={paragraph}
+                        highlighted={
+                            highlightedText && idx === paragraphs.length - 1 ? (
+                                <Highlighted highlightedText={highlightedText} />
+                            ) : null
+                        }
+                    />
+                );
+
+                return paragraphElement;
+            })}
+        </>
+    );
+};
+
+export const ContributionsEpic: React.FC<EpicProps> = ({
+    brazeMessageProps,
+    countryCode,
+    submitComponentEvent,
+}: EpicProps) => {
+    const [hasBeenSeen, setNode] = useHasBeenSeen({ threshold: 0 }, true) as HasBeenSeen;
+
+    useEffect(() => {
+        if (hasBeenSeen) {
+            // sendEpicViewEvent(tracking.referrerUrl, stage);
+        }
+    }, [hasBeenSeen]);
+
+    const cleanHighlighted = brazeMessageProps.highlightedText;
+
+    const cleanHeading = replaceNonArticleCountPlaceholders(brazeMessageProps.heading, countryCode);
+
+    const cleanParagraphs: string[] = [];
+    for (let i = 1; i < 10; i++) {
+        const paragraph: string = brazeMessageProps[`paragraph${i}`];
+        if (paragraph) {
+            cleanParagraphs.push(paragraph);
+        }
+    }
+
+    if (
+        [cleanHighlighted, cleanHeading, ...cleanParagraphs].some(
+            containsNonArticleCountPlaceholder,
+        )
+    ) {
+        console.warn('braze-components Epic does not support article count');
```

Comment:
I _think_ when the condition of this `if` block is met it means there are placeholders (like `%%SOMETHING%%`) in the text which _aren't_ related to `%%ARTICLE_COUNT%%`. I'm not sure why it's okay for `%%ARTICLE_COUNT%%` to still exist in the text, it seems to be handled separately than other placeholders.